src/main/java/harvard/Storage.java

```diff
@@ -0,0 +1,82 @@
+package harvard;
+
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * The Storage class is responsible for handling the storage of tasks in the Harvard application.
+ * It provides methods to load tasks from and store tasks to a text file.
+ */
+public class Storage {
+    /**
+     * The file path for storing tasks.
+     */
+    private String filePath;
+
+    /**
+     * Constructs a Storage instance with the specified file path.
+     *
+     * @param fP The file path for storing tasks.
+     */
+    private String filePath;
+    public Storage(String fP) {
+        this.filePath = fP;
+    }
+
+    /**
+     * Loads tasks from the stored text file and returns a BufferedReader.
+     * If the file is not found, it creates a new text file.
+     *
+     * @return The BufferedReader containing task information.
+     */
+    public BufferedReader load() {
```

Comment:
I like that you created the file storage location right at the start if the file is not detected instead of waiting till you save the save